### DISCUSSION TITLE
Status line when nvimtree is opened

### DISCUSSION THREAD

**Discussion by machinoro** [NvChad/NvChad#3103] [Q&A] [answered]
Hi, Im working on my config which is based heavily on nvchad config. However, I noticed that when I open nvimtree, the status line also displayed in the window and also when I split my window it will also display different statuslines for each window. Is there any way to solve that?

![image](https://github.com/user-attachments/assets/6c528bbb-af5a-4484-99fb-421181d43a7a)

**Comment by siduck** [comment, marked as answer]
set laststatus to 3 , remove the code which sets it to 2 from your config
